Fix FurthestAncestorOrSelfDynamicRootQueryStep and FurthestDescendantOrSelfDynamicRootQueryStep

erikjanwestendorp opened the pull request:
Fixes: https://github.com/umbraco/Umbraco-CMS/issues/15112 

The aliases of `FarthestAncestorOrSelfDynamicRootQueryStep` and `FarthestDescendantOrSelfDynamicRootQueryStep` were incorrect. Changed the aliases and also updated the class names to be consistent.

lauraneto commented:
Hi @erikjanwestendorp ,

Thank you for this fix! 🙌
I will see if I can get someone from HQ to have a look at it.

bergmania commented:
Dammit.. Frontend uses "Furthest" and backed uses "Farthest". 

Im pretty sure some integration tests will fail, if these are not updated with the current change in backend.

Whether we change frontend or backend, I don't care :)

erikjanwestendorp commented:
@bergmania You are absolutely right. I have just updated the integration test. I have also updated the method names in the repository to ensure consistent naming. In hindsight, it probably would have been faster to update the frontend. 😄

bergmania commented:
Thanks @erikjanwestendorp.. I was about to do the same thing with repository 💪

bergmania commented:
Correct, It's only breaking between the first RC and the final release
